docs/formats.md

```diff
@@ -52,64 +87,64 @@ Creates a CSS file with variable definitions based on the style dictionary
 
 * * *
 
-### scss/variables
+### scss/variables 
 
 
 Creates a SCSS file with variable definitions based on the style dictionary
 
-**Example**
+**Example**  
 ```scss
 $color-background-base: #f0f0f0;
 $color-background-alt: #eeeeee;
 ```
 
 * * *
 
-### scss/icons
+### scss/icons 
 
 
 Creates a SCSS file with variable definitions and helper classes for icons
 
-**Example**
+**Example**  
 ```scss
 $content-icon-email: '\E001';
 .icon.email:before { content:$content-icon-email; }
 ```
 
 * * *
 
-### less/variables
+### less/variables 
 
 
 Creates a LESS file with variable definitions based on the style dictionary
 
-**Example**
+**Color-background-base:**: #f0f0f0;  
```

Comment:
not sure about this change, is it correct? I see below the `@color` have been removed

Reply:
This is is a weird issue with jsdoc-2-md because our 'less' formats have an '@' in the example code which breaks the markdown file generation. Usually the doc generation script runs when we bump the version, at which point I will go and manually fix this part of the markdown file. If someone can figure out how to fix the issue I would be forever indebted to them.

Reply:
I'm thinking I should resolve this comment. Or is there a specific action we can take on it?